oaknational/oak-ai-lesson-assistant
# feat: allow for quiz exports to handle images

## Description

This allows for all of our doc exports to be able to handle export images. This work precedes the maths quiz questions work which will call on oak maths questions where the questions and answers are often in image form. The logic may also be reused in the future for adding images to slides.

How to test:
In order to test this run through the download process for a quiz and it should work with no changes
